db/seeds.rb

```diff
@@ -7,3 +7,175 @@
 #   Character.create(name: 'Luke', movie: movies.first)
 
 require 'faker'
+
+puts 'Destroying clients'
+Client.destroy_all
+
+puts 'Destroying users'
+User.destroy_all
+
+puts 'Destroying organizations'
+Organization.destroy_all
+
+puts 'Destroying venues'
+Venue.destroy_all
+
+puts 'Destroying events'
+Event.destroy_all
+
+puts 'Destroying orders'
+Order.destroy_all
+
+puts 'Destroying event categories'
+EventCategory.destroy_all
+
+puts 'Destroying orders'
+Venue.destroy_all
+
+puts 'Destroying tickets'
+Ticket.destroy_all
+puts 'Done!'
+
+# CREATE RECORDS
+
+# Create organizations
+
+15.times do
+  organization = Faker::Sports::Football.unique.team
+  website = Faker::Internet.domain_name
+
+  Organization.create!(name: organization, website: website)
+
+  puts "Created organization #{organization}"
+end
+
+puts 'Done creating organizations!'
+organizations = Organization.all
+
+# Create venues
+
+Venue.create!(name: 'Estádio Municipal Mário Wilson', address: 'Rua Coro de Santo Amaro de Oeiras 12')
+puts 'Estádio Municipal Mário Wilson added to venues'
+Venue.create!(name: 'Estádio José Gomes', address: 'Avenida Dom José Primeiro')
+puts 'Estádio José Gomes added to venues'
+Venue.create!(name: 'Estádio Municipal de Aveiro', address: 'Lugar Taboeira, Aveiro')
+puts 'Estádio Municipal de Aveiro added to venues'
+Venue.create!(name: 'Estádio Pina Manique', address: 'Estrada das Bungavilhas')
+puts 'Estádio Pina Manique added to venues'
+Venue.create!(name: 'Estádio Municipal do Fontelo', address: 'Avenida José Relvas 6, Viseu')
+puts 'Estádio Municipal do Fontelo added to venues'
+Venue.create!(name: 'Estádio Marcolino de Castro', address: 'Avenida Clube Desportivo Feirense')
+puts 'Estádio Marcolino de Castro added to venues'
+Venue.create!(name: 'Estádio Municipal de Mafra', address: 'Parque Desportivo Municipal de Mafra')
+puts 'Estádio Municipal de Mafra added to venues'
+Venue.create!(name: 'Estádio Municipal José dos Santos Pinto', address: 'Rua Sítio do Pinho do Gaiteiro 7, Covilhã')
+puts 'Estádio Municipal José dos Santos Pinto added to venues'
+Venue.create!(name: 'Estádio Municipal 25 de Abril', address: 'Rua Futebol Clube de Penafiel, Penafie')
+puts 'Estádio Municipal 25 de Abril added to venues'
+Venue.create!(name: 'Estádio de São Lúis', address: 'Rua do Sporting Club Farense, Faro')
+puts 'Estádio de São Lúis added to venues'
+
+puts "Done creating venues!"
+venues = Venue.all
+
+# Create clients
+
+100.times do
+  name = "#{Faker::Name.first_name} #{Faker::Name.last_name}"
+  email = Faker::Internet.email
+  phone = Faker::PhoneNumber.phone_number_with_country_code
+
+  Client.create!(
+    name: name,
+    email: email,
+    phone: phone
+  )
+
+  puts "Created client #{name}"
+end
+
+puts 'Done creating clients!'
+clients = Client.all
+
+# Create users
+
+35.times do
+  name = "#{Faker::Name.first_name} #{Faker::Name.last_name}"
+  email = Faker::Internet.email
+  role = rand(0..2)
```

Comment:
dont think we need to populate fake admins (check the enum in the User model ?)

+ instead populate ourselves as admin ?
thierry@tixfix.xyz
henrique@tixfix.xyz
pedro@tixfix.xyz
artur@tixfix.xyz